tools/os-settings/Windows/dependencies.cmake

```diff
@@ -20,3 +20,9 @@ include_directories(external/Freeglut/include)
 add_subdirectory(external/Freeglut ${CMAKE_BINARY_DIR}/external/Freeglut)
 link_directories("${CMAKE_BINARY_DIR}/external/Freeglut/lib")
 target_link_libraries(${PROJECT_NAME} PUBLIC freeglut)
+
+## GLEW
+add_subdirectory(external/glew ${CMAKE_BINARY_DIR}/external/glew)
+include_directories(external/glew/include)
+link_directories("${CMAKE_BINARY_DIR}/external/glew/lib")
+target_link_libraries(${PROJECT_NAME} PUBLIC libglew_shared)
```

Comment:
newline,
You need the same thing for linux

find_package(GLEW REQUIRED)
if (GLEW_FOUND)
include_directories($(GLEW_INCLUDE_DIRS))
endif()

target_link_libraries(${PROJECT_NAME} PUBLIC ${GLUT_LIBRARIES} ${GLEW_LIBRARIES})

Reply:
This is not finished yet.